### PR TITLE
[FIX] l10n_ma : fix external id

### PR DESCRIPTION
**Pull request by Bertrand2**
Fixes the external id that was using the kzc prefix instead of ma.

taskID 2930634

### CODE REVIEW

**Review by h4818 — APPROVED** [review]
l10n_ma_chart_template?

File: addons/hr_work_entry/models/hr_work_entry.py [hunk 1]
```diff
@@ -227,7 +227,6 @@ def _error_checking(self, start=None, stop=None, skip=False):
                 # no need to reload work entries.
                 work_entries.exists()._check_if_error()
 
-
```
[the remark this answers is not in the document]
Comment (Bertrand2) [reply addons/hr_work_entry/models/hr_work_entry.py]: no it seems not, not even sure how that slipped there. Thanks!

**Comment by h4818** [comment]
Should we not use the name `l10n_ma_chart_template`? so it doesn't sound like something temporary

File: addons/l10n_ma/data/account_chart_template_data.xml [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <odoo>
     <data noupdate="1">
         <function model="account.chart.template" name="try_loading">
-            <value eval="[ref('l10n_ma.l10n_kzc_temp_chart')]"/>
+            <value eval="[ref('l10n_ma.l10n_ma_chart_template')]"/>
```
Comment (tivisse) [inline addons/l10n_ma/data/account_chart_template_data.xml:5]: Requires a migration script.

**Comment by tivisse** [comment]
@robodoo r+

**Pull request closed without merging**